File: app/src/main/java/bj/discogsbrowser/model/search/RootSearchResponse.java

package bj.discogsbrowser.model.search;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import bj.discogsbrowser.model.common.Pagination;

public class RootSearchResponse
{
    @SerializedName("pagination")
    @Expose
    private Pagination pagination;
    @SerializedName("results")
    @Expose
    private List<SearchResult> searchResults = null;

    public Pagination getPagination()
    {
        return pagination;
    }

    public void setPagination(Pagination pagination)
    {
        this.pagination = pagination;
    }

    public List<SearchResult> getSearchResults()
    {
        return searchResults;
    }
}

File: app/src/main/java/bj/vinylbrowser/model/search/RootSearchResponse.kt
package bj.vinylbrowser.model.search

import bj.vinylbrowser.model.common.Pagination
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class RootSearchResponse(val pagination: Pagination, @SerializedName("results") val searchResults: List<SearchResult>)